playground/bicep/BicepSample.AppHost/aspire.hosting.azure.bicep.cosmosdb.bicep

```diff
@@ -10,32 +11,43 @@ param location string = resourceGroup().location
 var resourceToken = uniqueString(resourceGroup().id)
 
 resource cosmosDb 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' = {
-  name: replace('${databaseAccountName}-${resourceToken}', '-', '')
-  location: location
-  kind: 'GlobalDocumentDB'
-  tags: tags
-  properties: {
-    consistencyPolicy: { defaultConsistencyLevel: 'Session' }
-    locations: [
-      {
-        locationName: location
-        failoverPriority: 0
-      }
-    ]
-    databaseAccountOfferType: 'Standard'
-  }
-
-  resource db 'sqlDatabases@2023-04-15' = [for name in databases:{
-    name: '${name}'
+    name: replace('${databaseAccountName}-${resourceToken}', '-', '')
```

Comment:
Given this resource name is scopes to the database we probably don't need the resource token here. Additionally for cosmos specifically it could be a problem because you need to embed the database name somewhere (env or in the code).

Reply:
My bad. i was trying to review this on my mobile whilst the power was out. Its clearer now :)